src/libfsm/print/cdata.c

```diff
@@ -1044,6 +1320,14 @@ populate_config_from_ir(struct cdata_config *config, const struct fsm_alloc *all
 		si->eager_output = STATE_OFFSET_NONE;
 	}
 
+	/* add a single entry for 0, in case the IR only has a single IR_NONE or IR_SAME state */
+	config->bitset_words.ceil = 1;
+	config->bitset_words.used = 1;
+	config->bitset_words.pairs = calloc(1, sizeof(config->bitset_words.pairs[0]));
```

Comment:
missing error handling here, and also this should be `f_calloc()`

Reply:
Fixed in a99b9137, now the caller also checks the return value and returns `-1` on error, as per the print interface.